### ISSUE TITLE
Implement detailed results page

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#107]

**Comment by neslihanu** [comment]
Notes for the UX designers;

- How to handle "Citations" feature; check FreeCen and FreeCEN pages and Pat`s message about this feature is copied below:
yes, we are planning to keep Citations.  It will be a nice addition to FreeBMD2.  We want to include it (and one more 'nice to have' in the MVP definition so that experienced users of FreeBMD1 get something additional on FreeBMD2, not just the pain of having to use a new layout, etc.  There is a 'citations' on FreeBMD1, but this is not formatted for the different types of family tree programmes or academic citations, as the  multi-modal ones are, plus it does not have the 'copy to clipboard' feature (currently on test3 only for FreeCEN and FreeREG - rather than opening in a new page for a manual copy and paste, clicking the button copies to clipboard.

**Comment by neslihanu** [comment]
Please find the draft design for Search Results Detail Page (Marriages). I`ve considered below points during the design:

- I don`t want to delete any feature such as "adding a postem" or "submitting a correction". You can view the Freebmd1 search results detail page by clicking this link: https://www.freebmd.org.uk/cgi/information.pl?r=201674369:1144&d=bmd_1564415191

- For citations layout, I designed in the same way as  FreeCEN and FreeREG. We may have a look at it again and consider a new design. 

- I just copied and pasted the texts from FreeBMD1, if you have any updates please share the last version of the text with me. 

- "Spouses name" header on the table should be changed according to the record type. 

![01.00_Search Results Detail Page_Marriages (1).png](https://images.zenhubusercontent.com/5cd985d5d341f7297326ca0e/2a929188-22ea-4702-af08-751f1db9e78f)

**Comment by neslihanu** [comment]
Please find the updated mockup below. I`ve changed the icons to fontawesome icons and made some small updates. 

![01.00_Search Results Detail Page_Marriages.v2.png](https://images.zenhubusercontent.com/5cd985d5d341f7297326ca0e/7a75e792-08a5-477f-bfaf-a74acb7089a8)

**Comment by PatReynolds** [comment]
I think some icons/associated information would be better located on this page (detailed results page) than on the results page.

Ie the results page is for information which will allow the user to differentiate between the various records (e.g. names differ, locations differ, etc.) and other information is in the detailed results.

I am undecided on the 'view' - I think that might be needed in both.

System entry - leave 'new entry'  on results page
System link and system entry - move to detailed results page 

Comments:  comment and postem - only on detailed

Feedback needed.

**Comment by neslihanu** [comment]
@PatReynolds I totally agree with you. When we started to redesign Search Results Page, I and Aviel talked about this but then we had a limited schedule to create the mock-up (2 weeks for the first version) so we did not want to remove any feature/function/icon without having a technical discussion. 
I believe it would be good to start a discussion about those icons and their functions. During designing, I tried to find an entry that has a system link or system entry, I could not find any manually. Maybe it would be good to check the DB and see that how many records have those info.

Today, do we really need all of them ? Do FreeBMD1 users use them ?

Also I`m adding @richpomfret to keep him posted.

**Comment by PatReynolds** [comment]
Date, Quarter, Record Type, District, Vol/page need to follow one another.

**Comment by archerbarrie** [comment]
On Oct 28 @PatReynolds  wrote:
>I think some icons/associated information would be better located on this page (detailed results page) than on the results page.

>Ie the results page is for information which will allow the user to differentiate between the various records (e.g. names differ, locations differ, etc.) and other information is in the detailed results.

>I am undecided on the 'view' - I think that might be needed in both.

>System entry - leave 'new entry' on results page
System link and system entry - move to detailed results page

>Comments: comment and postem - only on detailed

I am struggling to see how this is substantially different from FreeBMD1. On F1 we use icons on the results page to indicate that the Information page has additional information, specifically postems, comments and references to other records. One can also see the transcriber id (pretty useless information) and can submit a correction; there are no icons associated with these.

The one difference I can see is that some (unspecified) icons will only be on the Information page. I cannot really see the point of that.

**Comment by archerbarrie** [comment]
On 28 Oct @neslihanu wrote:

>During designing, I tried to find an entry that has a system link or system entry, I could not find any manually. Maybe it would be good to check the DB and see that how many records have those info.

There are just over 5000 system entries and there should be about the same number of links (I think there can be multiple links for one system entry). An example is https://www.freebmd.org.uk/cgi/information.pl?cite=EWLDmx8t2z6R7qu4RoulVg&scan=1

**Comment by archerbarrie** [comment]
On 28 Oct @neslihanu wrote:

>Today, do we really need all of them ? Do FreeBMD1 users use them ?

This is a question about system entries

 I don't understand why we would want to drop some of them ... in any case we would still need the facility for the rest.

A number of system entries were requested by users (e.g. to correct their own entry).

So, the answers are Yes and Yes.

**Comment by archerbarrie** [comment]
on 27 Sep @neslihanu provided a mocked up screenshot of the Information page. It would be better if a wireframe was provided but with the limited information in a screenshot I have the following comments.

The "next step" may be to order a certificate but almost certainly is not.

There may be several entries that went to make up the search result, however there is only one scan so the scan icon should not be attached to each entry but to page as a whole.

The scan icon needs to take the user to another page to select which scan to view (if there are multiple scans) and to select the format. In FreeBMD1 if you click on the scan icon in the search results this information is displayed on the information page. With this mock up an extra click and page is required.

One really needs a separate mock up for each class of entry, it is not just that the Spouse name column changes.

Why does the Spouse name come between the district and the volume? That is an odd order.

What does Previous and Next entry mean? Is it the next or previous entry on the page or in the search results. If the latter how long do we cache the search results so we can do this?

**Comment by KatyMawhood** [comment]
@richpomfret can this be closed and linked into the epic [Search experience #227](https://github.com/FreeUKGen/FreeBMD2/issues/227)?

**Comment by janetkwakye-afk** [comment]
We have reviewed this epic and confirmed that it meets the agreed requirements. It can therefore be closed.
